client/src/Pages/Cart/Cart.jsx

```diff
@@ -0,0 +1,105 @@
+import React from "react";
+import { CartCard } from "./CartCard/CartCard";
+import { CheckOutHeader } from "../../components/CheckOutHeader/CheckOutHeader";
+import { useReducerContext } from "../../Context/ReducerContext";
+import bag from "../../assets/bag.png";
+import { Link } from "react-router-dom";
+
+export function Cart() {
+    const { cart } = useReducerContext();
+
+    function getCartQuantity() {
+        return cart.reduce(
+            (totalItems, item) => totalItems + 1 * item.quantity,
+            0
+        );
+    }
+
+    function getTotalAmount() {
+        return cart.reduce(
+            (totalPrice, item) =>
+                totalPrice + item.quantity * item.discountedPrice,
+            0
+        );
+    }
+
+    function getTotalAmountBeforeDiscount() {
+        return cart.reduce(
+            (totalPrice, item) => totalPrice + item.quantity * item.price,
+            0
+        );
+    }
+
+    return (
+        <div className="cart-page">
+            <CheckOutHeader heading={"Cart"} step={"Step 1/3"} />
+            <div className="cart-page-body">
+                <div className="cart-page-body-overview-and-products">
+                    {cart.length !== 0 ? (
+                        <div className="cart-overview">
```

Comment:
this cart page looked little different for me when I opened, Like the navbar lost the icons and I can see the text decoration for the logo here , it would be good if u disable the decor  I think